vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/DefineExperimentComponent.java

```diff
@@ -28,9 +29,8 @@
  *
  * @since 1.0.0
  */
-public class DefineExperimentComponent extends Composite<VerticalLayout> {
+public class DefineExperimentComponent extends Div {
```

Comment:
I think we need a `@Tag(Tag.DIV)` annotation here, otherwise it wont be treated as native HTML div element

Reply:
please ignore, since we extend `Div`, no need to add the tag explicitly (it is done in the `Div.class` already)